MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -367,7 +367,8 @@ public Task<MediaInfo> GetMediaInfo(MediaInfoRequest request, CancellationToken
         public string GetInputArgument(string inputFile, MediaSourceInfo mediaSource)
         {
             var prefix = "file";
-            if (mediaSource.VideoType == VideoType.BluRay)
+            if (mediaSource.VideoType == VideoType.BluRay ||
+                mediaSource.IsoType == IsoType.BluRay)
```

Comment:
For consistency with the code base
```suggestion
            if (mediaSource.VideoType == VideoType.BluRay
                || mediaSource.IsoType == IsoType.BluRay)
```